pointerevents/pointerlock/pointerevent_movementxy-manual.html

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Pointer Events properties tests</title>
+        <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" type="text/css" href="../pointerevent_styles.css">
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <!-- Additional helper script for common checks across event types -->
+        <script type="text/javascript" src="../pointerevent_support.js"></script>
+        <style>
+          body {
+            touch-action: none;
+            user-select: none;
+          }
+          #box1 {
+            top: 330px;
+            left: 150px;
+            background: black;
+          }
+          #box2 {
+            top: 370px;
+            left: 350px;
+            background: red;
+          }
+          #square2 {
+            visibility: block;
+          }
+        </style>
+        <script>
+            var expectedPointerId = NaN;
+            var startSummation = false;
+            var isFirstMove = true;
+            var lastClientX = 0;
+            var lastClientY = 0;
+
+            function resetTestState() {
+                startSummation = false;
+                isFirstMove = true;
+                lastClientX = 0;
+                lastClientY = 0;
+            }
+
+            function run() {
+                var test_pointerEvent = setup_pointerevent_test("pointerevent attributes", ['mouse', 'touch']);
+
+                [document, document.getElementById('innerFrame').contentDocument].forEach(function(element) {
+                  on_event(element, 'pointermove', function (event) {
+                    if (startSummation) {
+                      if (!isFirstMove) {
+                          test_pointerEvent.step(function() {
+                            assert_equals(event.movementX, event.clientX - lastClientX, "movementX should be the delta between current event's and last event's clientX");
+                            assert_equals(event.movementY, event.clientY - lastClientY, "movementY should be the delta between current event's and last event's clientY");
+                          });
+                      }
+                      isFirstMove = false;
+                      lastClientX = event.clientX;
+                      lastClientY = event.clientY;
+                    }
+                  });
+                });
+                ['pointerenter', 'pointerleave'].forEach(function(eventName) {
+                  on_event(document.getElementById('innerFrame').contentDocument, eventName, function (event) {
+                    isFirstMove = true;
```

Comment:
screenX/Y are good, the use of clientX/Y in other tests was an oversight. I've filed an issue to fix the pointerlock tests #4630.